lib/services/android-project-service.ts

```diff
@@ -143,24 +140,12 @@ class AndroidProjectService extends projectServiceBaseLib.PlatformProjectService
 
 			let gradleSettingsFilePath = path.join(this.platformData.projectRoot, "settings.gradle");
 			shell.sed('-i', /__PROJECT_NAME__/,  this.$projectData.projectName, gradleSettingsFilePath);
+			shell.sed('-i', /__APILEVEL__/, this.$options.sdk || this.$androidToolsInfo.getToolsInfo({showWarningsAsErrors: true}).wait().compileSdkVersion.toString(), manifestPath);
```

Comment:
We could, but as the result is cached inside getToolsInfo, we'll get the cached result... So we'll have two cached variables with the same value